prompts/template_xml/triage_problem.txt

```diff
@@ -0,0 +1,12 @@
+Below is the crash report:
+<log>
+{CRASH_REPORT}
+</log>
+
+Below is the fuzz driver:
+<code>
+{FUZZ_TARGET_CODE}
+</code>
+
+Based on the above information, analyze the cause of the crash. First, only answer “the crash is caused by the bug in the fuzz driver” or “the crash is caused by the bug in the project”. Second, give analysis.
+<solution>
```

Comment:
I reckon lines 11-12 are general requirements, hence belong to priming?
This `problem` should be a single problem.

Imagine we will have some example triage problem-solution pairs later: This template will be used to formulate the sample problem, and we don't need to repeat this in each sample problem.

Reply:
nit: s/fuzz driver/fuzz target/g

Reply:
Sorry, I didn't fully understand what you meant, especially "nit: s/fuzz driver/fuzz target/g". The following are modifications based on my understanding:

priming:
https://github.com/fdt622/oss-fuzz-gen/raw/dev/prompts/template_xml/triager_priming.txt

problem:
https://github.com/fdt622/oss-fuzz-gen/raw/dev/prompts/template_xml/triager_problem.txt

If other modifications are needed or I have misunderstood something, please point it out.